modules/ROOT/pages/munit-maven-plugin.adoc

```diff
@@ -553,16 +553,15 @@ MUnit allows you to skip the rest of the tests if one test suite fails:
 </plugins>
 ----
 
-If not specified, this value is `false` by default.
-
+If not specified, it's default value is `false`.
 
 [[specify-runtime-product]]
-== Specify The Runtime Product
+== Specify the Runtime Product
 
-You can specify the type of runtime in which your applications being tested will run. Accepted values are:
+You can specify the type of runtime in which the applications you test run. Accepted values are:
 
-* `MULE` for community edition.
-* `MULE_EE` for enterprise edition.
+* `MULE` for the community edition.
+* `MULE_EE` for the enterprise edition.
```

Comment:
Mule EE and Mule CE have specific names. 

https://docs.mulesoft.com/mule-runtime/latest/installing-an-enterprise-license

I think CE is now called Mule Kernel. Need to check.

Reply:
https://docs.mulesoft.com/mule-runtime/latest/mule-4-api-javadoc

Reply:
Great catch! Thanks!